src/Raven.Client/Documents/Operations/Revisions/RevisionsCollectionConfiguration.cs

```diff
@@ -15,13 +15,16 @@ public sealed class RevisionsCollectionConfiguration
 
         public long? MaximumRevisionsToDeleteUponDocumentUpdate { get; set; }
 
+        public bool AllowDeleteRevisionsManually { get; set; }
```

Comment:
Changes to the revision are currently logged to the audit log?

Reply:
Do you mean changes for the revisions configuration?

Reply:
Yes, we need to add that to the audit log, to ensure that there is a record on when we allowed that.

May also want to add a `AuditManualRevisionDeletion` or something like that, so each such write would also be logged.

Reply:
In fact, every time we change the revisions configuration, it is logged to the audit log.